lib/autoload/course/assessment/java/java_programming_test_case_report.rb

```diff
@@ -184,8 +192,8 @@ def status
     def messages
       # prune empty and nil values
       @messages ||= {
-        'error': nil,
-        'error_contents': nil,
+        'error': failure_message,
+        'error_contents': failure_contents,
```

Comment:
Quick question: will the `failure_contents` is exceptionally large in certain cases? Since it looks like it is outputting the entire stacktrace. Any idea how that affects our front end?